obspy/io/arclink/inventory.py

```diff
@@ -0,0 +1,908 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+ObsPy implementation for parsing the arclink inventory format
+to an Inventory object.
+
+This is a modified version of obspy.io.stationxml and obspy.io.sc3ml.
+
+:author:
+    Mathijs Koymans (koymans@knmi.nl), 29.2016 - [Jollyfant@GitHub]
+:copyright:
+    The ObsPy Development Team (devs@obspy.org)
+:license:
+    GNU Lesser General Public License, Version 3
+    (http://www.gnu.org/copyleft/lesser.html)
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+
+import inspect
+import re
+import math
+import os
+import warnings
+import obspy
+
+from lxml import etree
+from obspy.core.util.obspy_types import (ComplexWithUncertainties,
+                                         FloatWithUncertaintiesAndUnit)
+from obspy.core.inventory import (Azimuth, ClockDrift, Dip,
+                                  Distance, Frequency, Latitude,
+                                  Longitude, SampleRate)
+from obspy.core.inventory import (CoefficientsTypeResponseStage,
+                                  FilterCoefficient, FIRResponseStage,
+                                  PolesZerosResponseStage,
+                                  PolynomialResponseStage)
+
+SOFTWARE_MODULE = "ObsPy %s" % obspy.__version__
+SOFTWARE_URI = "http://www.obspy.org"
+SCHEMA_VERSION = "1.0"
+SCHEMA_NAMESPACE = "http://geofon.gfz-potsdam.de/ns/Inventory/1.0/"
+
+
+def _is_inventory_xml(path_or_file_object):
+    """
+    Simple function checking if the passed object contains a valid arclink XML
+    1.0 file. Returns True of False.
+    The test is not exhaustive - it only checks the root tag but that should
+    be good enough for most real world use cases. If the schema is used to
+    test for a StationXML file, many real world files are false negatives as
+    they don't adhere to the standard.
+    :param path_or_file_object: File name or file like object.
```

Comment:
Please put a blank line before `params` block. In general we also have a blank line after the first line (in this case after the first two lines) of the docstring.